src/commonMain/kotlin/exchange.dydx.abacus/processor/router/skip/SkipTrackProcessor.kt

```diff
@@ -12,8 +12,11 @@ internal class SkipTrackProcessor(
         payload: Map<String, Any>
     ): Map<String, Any>? {
         val modified = existing?.mutable() ?: mutableMapOf()
-        val txHash = parser.asString(payload.get("tx_hash")) ?: return modified
-        modified[txHash] = true
+        var txHash = parser.asString(payload.get("tx_hash")) ?: return modified
+        if (!txHash.startsWith("0x")) {
+            txHash = "0x$txHash"
```

Comment:
i'm not sure we always want this though, what if the user submits a non EVM transaction? the tx hash may not necessarily start with 0x, right? 
@ruixhuang